_studio/mfx_lib/scheduler/src/mfx_scheduler_core_thread.cpp

```diff
@@ -135,6 +134,7 @@ void mfxSchedulerCore::WakeupThreadProc()
 
             //MFX_AUTO_LTRACE(MFX_TRACE_LEVEL_SCHED, "HW Event");
             IncrementHWEventCounter();
+            std::lock_guard<std::mutex> guard(m_guard);
```

Comment:
Please, add {} around lock and protected function. This emphasize what you actually protect.